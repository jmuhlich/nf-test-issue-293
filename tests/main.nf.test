nextflow_pipeline {

    name "Test pipeline"
    script "../main.nf"

    test("first test") {
        then {
            assert snapshot([]).match()
        }
    }

}
