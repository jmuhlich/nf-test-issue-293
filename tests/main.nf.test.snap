{
    "first test": {
        "content": [
            [
                "foo.txt"
            ]
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.6"
        },
        "timestamp": "2025-09-25T01:16:12.785722811"
    }
}
